### PR TITLE
Update some vs. docs

### PR DESCRIPTION
**Pull request by cnunciato**
Updates our vs. CloudFormation and vs. CDK docs.

### CODE REVIEW

**Review by gthuang — APPROVED** [review]

File: themes/default/content/docs/intro/vs/aws-cdk/_index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Finally, CDK and Pulumi have similar concepts for modularity or higher-level abstractions that encapsulate multiple resources. These are known as constructs in CDK and component resources in Pulumi.
+
+## Pulumi vs. CloudFormation: Key Differences {#differences}
+
+First, CDK only supports AWS, while Pulumi supports over 60 cloud and SaaS providers. Second, although both CDK and Pulumi support languages like TypeScript, Python, Go, and .NET, the tools are fundamentally different in how they interpret the languages and deploy resources. Unlike Pulumi, whose open-source engine understands these languages, CDK is a transpiler (a.k.a., source-to-source compiler) that translates programs into AWS CloudFormation YAML/JSON. The resulting markup file is then submitted to the closed-source AWS CloudFormation service, which provides the deployment engine for provisioning resources.
```
Comment (jaxxstorm) [inline themes/default/content/docs/intro/vs/aws-cdk/_index.md:38]: Would be good to add the fact we're adding more providers every day (and growing)

Comment (jaxxstorm) [inline themes/default/content/docs/intro/vs/aws-cdk/_index.md:38]: There are some caveats to this which has irked CDK proponents so far. We should consult https://github.com/pulumi/pulumi-hugo/issues/1275

File: themes/default/content/docs/intro/vs/aws-cdk/_index.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+| Feature | Pulumi | AWS CDK |
+| ------- | ------ | -------------- |
+| [Language Support](#language) | TypeScript, JavaScript, Python, Go, C#, F#, VB.NET | Python, TypeScript, JavaScript, Go (developer preview), C#, Java |
+| [IDE Support](#ide) | IntelliSense, code completion, strong typing, error squiggles, rich documentation, interactive debugging, and more | Code completion, type safety, inline documentation, etc. |
+| [State Management](#state) | Managed by Pulumi Service or with self-managed options | Managed by the AWS CloudFormation service |
+| [Provider Support](#providers) | Supports all major cloud providers, including 100% same-day coverage of new resources with AWS Native | AWS only |
+| [Cloud Native Support](#cloud-native) | Richly typed support for the full cloud-native ecosystem and Kubernetes | AWS only |
+| [Dynamic Provider Support](#dynamic-providers) | Yes | Limited |
+| [OSS License](#license) | Apache License 2.0 | Apache License 2.0 (only applies to the CD framework, and not the deployment engine) |
```
Comment (gthuang) [inline themes/default/content/docs/intro/vs/aws-cdk/_index.md:58]:
```suggestion
| [OSS License](#license) | Apache License 2.0 | Apache License 2.0 (only applies to the CDK framework and not the deployment engine) |
```

**Comment by cnunciato** [comment]
I'm going to merge this despite the build failure because the failure is not related to anything in this PR, and it needs to go out.

**Pull request merged**